### PR TITLE
Add random storage prefix name

### PR DESCRIPTION
**Pull request by lborguetti**
We need create random prefix name to Azure Storage Account
@katcipis @tiago4orion make sense? 

This is a recommendation by @fabioharams

### CODE REVIEW

**Comment by katcipis** [comment]
What is the rationale behind the recommendation ?

**Comment by lborguetti** [comment]
@katcipis 

> **Partition Naming Convention**
> 
> Azure Storage uses a range-based partitioning scheme to scale and load balance the system. The partition key is used to partition data into ranges and these ranges are load-balanced across the system. This means naming conventions such as lexical ordering (e.g. msftpayroll, msftperformance, msftemployees, etc) or using time-stamps (log20160101, log20160102, log20160102, etc) will lend itself to the partitions being potentially co-located on the same partition server, until a load balancing operation splits them out into smaller ranges. For example, all blobs within a container can be served by a single server until the load on these blobs requires further rebalancing of the partition ranges. Similarly, a group of lightly loaded accounts with their names arranged in lexical order may be served by a single server until the load on one or all of these accounts require them to be split across multiple partitions servers. Each load balancing operation may impact the latency of storage calls during the operation. The system’s ability to handle a sudden burst of traffic to a partition is limited by the scalability of a single partition server until the load balancing operation kicks-in and rebalances the partition key range.
> 
> You can follow some best practices to reduce the frequency of such operations.
> 
> Examine the naming convention you use for accounts, containers, blobs, tables and queues, closely. Consider prefixing account names with a 3-digit hash using a hashing function that best suits your needs.

ref: https://docs.microsoft.com/en-us/azure/storage/storage-performance-checklist

File: azure/storage.sh [hunk 1]
```diff
@@ -1,14 +1,15 @@
 # Storage related functions
 
 fn azure_storage_account_create(name, group, location, sku, kind) {
```
Comment (tiago4orion) [inline azure/storage.sh]: I think we must document the function saying that `name` will be used as suffix of a random name. And maybe point the user to azure documentation regarding storage locality.

Comment (tiago4orion) [inline azure/storage.sh]: And, this is the first function of `azure` library that returns a resource id. In every other function the resource id is managed by user.

Comment (lborguetti) [inline azure/storage.sh]: Agree :+1:

**Comment by tiago4orion** [comment]
Owww! You don't need to document everything now! 
I meant to add docs only in the storage functions...

**Comment by tiago4orion** [comment]
LGTM!

Can we merge?

**Comment by lborguetti** [comment]
Yes we can! :+1:

**Pull request merged**